### PR TITLE
Style Body and Add Colours for Theme

### LINKED ISSUE #136: Style Document Body

### PR DESCRIPTION
**Pull request by samuelsogbesan**
This PR integrates the basic styling required for the <code>&lt;body></code>.

- **Sets CSS Variables for theme colouring**
- **Set background colour**
- **Sets base text colour**

Closes #136

### CODE REVIEW

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  --background: #0C0D25;
+  --primary: white;
+  font-size: 16px;
+}
+
+body {
+  background-color: var(--background);
+  color: var(--primary);
+  display: flex;
+  font-family: 'Roboto', sans-serif;
```
Comment (MagoDelBlocco) [inline src/main/webapp/style.css:13]: please use the shorthand [font](https://developer.mozilla.org/en-US/docs/Web/CSS/font) property, here and above

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  flex-direction: column;
+  margin: 0.5rem 3rem;
+  overflow-y: scroll;
```
Comment (MagoDelBlocco) [inline src/main/webapp/style.css:16]: please use the shorthand [overflow](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow) property

**Review by MagoDelBlocco — CHANGES_REQUESTED** [review]

File: src/main/webapp/style.css (hunk 2 above)
Comment (samuelsogbesan) [reply src/main/webapp/style.css]: Implemented shorthand property 476a936

File: src/main/webapp/style.css (hunk 1 above)
Comment (samuelsogbesan) [reply src/main/webapp/style.css]: Replaced with shorthand e13de13

File: src/main/webapp/style.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap);
+
+:root {
+  --background: #0C0D25;
+  --primary: white;
+  font-size: 16px;
+}
+
+body {
+  background-color: var(--background);
```
Comment (MagoDelBlocco) [inline src/main/webapp/style.css:10]: please use the shorthand property `background` here as well

**Review by MagoDelBlocco — APPROVED** [review]
LGTM

File: src/main/webapp/style.css (hunk 3 above)
Comment (samuelsogbesan) [reply src/main/webapp/style.css]: Replaced here b5fc725

File: src/main/webapp/style.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  --primary: white;
+  font-size: 16px;
+}
+
+body {
+  background: var(--background);
+  color: var(--primary);
+  display: flex;
+  font: 1rem normal 'Roboto' sans-serif;
+  flex-direction: column;
```
Comment (Somae) [inline src/main/webapp/style.css:14]: Should this be ordered alphabetically?

**Review by Somae — CHANGES_REQUESTED** [review]

**Comment by Somae** [comment]
Please consider adding short comments explaining the result of the CSS styling class as documentation / overview.

File: src/main/webapp/style.css (hunk 4 above)
Comment (samuelsogbesan) [reply src/main/webapp/style.css:13]: Reordered here 1bab457

**Comment by samuelsogbesan** [comment]
> Please consider adding short comments explaining the result of the CSS styling class as documentation / overview.

Added a few clarifying comments.

**Review by Somae — APPROVED** [review]

**Pull request merged**